src/WorldIDIdentityManagerImplV1.sol

```diff
@@ -159,6 +162,13 @@ contract WorldIDIdentityManagerImplV1 is OwnableUpgradeable, UUPSUpgradeable, IW
     ///         initialized.
     error ImplementationNotInitialized();
 
+    /// @notice Thrown when attempting to send a transaction to the state bridge proxy that fails.
+    error StateBridgeProxySendRootMultichainFailure();
+
+    /// @notice Thrown when attempting to send a transaction to the state bridge proxy when the proxy address
+    /// is set to 0x0 (the zero address) indicating that the state bridge has been turned off. 
+    error stateBridgeHasBeenTurnedOff();
```

Comment:
We absolutely don't want to error in this case. My intention with our earlier discussion was that we would be able to transparently turn off the state bridge if we needed to, and things would continue without it.